core/src/main/java/org/elasticsearch/index/query/TermQueryParser.java

```diff
@@ -32,6 +32,8 @@
 
     private static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of term query");
     private static final ParseField BOOST_FIELD = new ParseField("boost").withAllDeprecated("boost is not supported in short version of term query");
```

Comment:
Maybe rename NAME_FIELD and BOOST_FIELD here as well, since they are only here to support the deprecated short version.